Sources/Terminal/AngouriMath.Terminal.Lib/FSharpInteractive.fs

```diff
@@ -70,29 +66,20 @@ let execute (kernel : FSharpKernel) code =
     | Some res -> res
 
 
-let private loadAssembly kernel (path : string) =
-    path.Replace("\\", "\\\\")
-    |> (fun loc -> execute kernel $"#r \"{loc}\"")
+
 
 
 let createKernel () =
     let kernel = new FSharpKernel ()
 
-    let load (typeInfo : Type) =
-        let assemblyLocation = typeInfo.Assembly.Location
-        if System.IO.File.Exists assemblyLocation then
-            match loadAssembly kernel assemblyLocation with
-            | Error error -> Result.Error error
-            | _ -> Result.Ok ()
-        else
-            Result.Error $"Assembly {assemblyLocation} does not exist"
-
-
-    load typeof<MathS>
-    |> Result.bind (fun _ -> load typeof<AngouriMath.FSharp.Core.ParseException>)
-    |> Result.bind (fun _ -> load typeof<AngouriMath.InteractiveExtension.KernelExtension>)
-    |> Result.bind (fun _ -> load typeof<Plotly.NET.Chart>)
-    |> Result.bind (fun _ ->
+    let assemblyLoad = AssemblyLoadBuilder (execute, kernel)
+
+    assemblyLoad {
+        yield! typeof<AngouriMath.MathS>
```

Comment:
F# has implicit yields and if you use `yield` you can omit all the `yield!`s

Reply:
Can you elaborate on it?

Reply:
If you use `Yield` instead of `YieldFrom`, you can omit the entire `yield` keyword

Reply:
Nice 😍